src/lib.ts

```diff
@@ -562,3 +564,76 @@ export function refillLatte(): void {
   const lastIngredient = get("latteUnlocks").includes("carrot") ? "carrot" : "pumpkin";
   if (get("_latteRefillsUsed") < 3) cliExecute(`latte refill cinnamon vanilla ${lastIngredient}`);
 }
+
+export function statToMaximizerString(stat: Stat): string {
+  return stat === $stat`Muscle` ? "mus" : stat === $stat`Mysticality` ? "myst" : "mox";
+}
```

Comment:
It doesn't appear that we are ever calling `statToMaximizerString()` with an argument other than `myPrimestat()`. If so, why don't we just directly `export const mainStatMaximizerString` instead of making it a function?

Reply:
```ts
export const mainStatMaximizerString = myPrimestat() === $stat`Muscle` ? "mus" : myPrimestat() === $stat`Mysticality` ? "myst" : "mox";
```

Reply:
Resolved.